lib/battle/CBattleInfoCallback.cpp

```diff
@@ -1552,23 +1553,19 @@ SpellID CBattleInfoCallback::getRandomBeneficialSpell(CRandomGenerator & rand, c
 		case SpellID::SHIELD:
 		case SpellID::FIRE_SHIELD: // not if all enemy units are shooters
 		{
-			auto walker = getAliveEnemy([&](const CStack * stack) //look for enemy, non-shooting stack
-			{
-				return !stack->canShoot();
-			});
+				const auto * walker = getAliveEnemy([&](const CStack * stack) //look for enemy, non-shooting stack
+													{ return !stack->canShoot(); });

```

Comment:
Formatting. Original code used correct style for lambda's inside method call

Also - alignment of entire block is 2 indents deep compared to brackets